RELEASE.md

```diff
@@ -1,3 +1,24 @@
+# Release 1.1.0
+
+## New features
+
+- New field `numberOfVectors` in the response of the `get_stats` endpoint (https://github.com/marqo-ai/marqo/pull/553). 
+This field indicates the number of vectors in an index so that users can monitor the status of their index. See [here](https://docs.marqo.ai/1.1.0/API-Reference/stats/) for detailed usage.
```

Comment:
Remove "so that users can monitor the status of their index." (As this statement doesn't add much new meaning)

You can make a comment like "this counts all vectors from all documents in this index" (giving users a clue about how it is calculated)